packages/headless/package.json

```diff
@@ -12,8 +12,8 @@
   "version": "0.15.2",
   "files": [
     "dist/",
-    "case-assist/",
```

Comment:
I removed `case-assist` from the npm tarball since the use-case is still not implemented. I did not want to give a false impression that it was available,